Telcodocs 338 Configuring hub cluster to scale up to 2000 cluster (#42424)
openshift/openshift-docs: modules/ztp-creating-ztp-config-hub-for-scaling-to-2000.adoc

// Module included in the following assemblies:
//
// *scalability_and_performance/ztp-deploying-disconnected.adoc

:_content-type: PROCEDURE
[id="ztp-configuring-hub-clusters-for-scaling-to-2000_{context}"]
= Configuring the hub clusters for scaling to 2000 cluster deployments

> The following procedure tells you how to configure a hub cluster at scale to deploy and manage up to 2000 managed single node hub clusters configured as distributed units (DUs).

Reviewer: hub cluster managing spoke clusters I think.

```suggestion
You can configure a hub cluster to deploy and manage up to 2000 managed single node spoke clusters configured as distributed units (DUs).
```
